kernelci/build.py

```diff
@@ -802,6 +802,28 @@ def _kernel_config_enabled(self, config_name):
         cmd = 'grep -cq CONFIG_{}=y {}'.format(config_name, dot_config)
         return shell_cmd(cmd, True)
 
+    def _kernel_config_getkey(self, opt_name):
+        dot_config = os.path.join(self._output_path, '.config')
+        with open(dot_config, 'r') as fp:
+            lines = fp.readlines()
+            for line in lines:
+                pcfg = line.strip().split('=')
+                if len(pcfg) > 1 and pcfg[0] == opt_name:
+                    return pcfg[1]
+        return False
+
+    def _kernel_config_setkey(self, opt_name, opt_value):
+        dot_config = os.path.join(self._output_path, '.config')
+        dot_confignew = os.path.join(self._output_path, '.confignew')
+        with open(dot_config, 'r') as fr, open(dot_confignew, 'w') as fw:
+            lines = fr.readlines()
+            for line in lines:
+                pcfg = line.strip().split('=')
+                if len(pcfg) > 1 and pcfg[0] == opt_name:
+                    line = f"{opt_name}={opt_value}\n"
+                fw.write(line)
+        os.replace(dot_confignew, dot_config)
```

Comment:
Using `./scripts/config --enable CONFIG_SOMETHING` would be much better imho.

Reply:
Will do, i thought we need to avoid unnecessary use of subprocess :)

Reply:
Here it's necessary as if kernel configs need to be done differently in different kernel versions then calling the `scripts/config` script will always work (in theory).

Reply:
Done, probably in near future getkey and there is nearby option will be updated too, but likely in separate commit, as i think we need to get firmware embedding work properly so we can move further with solving other ChromiumOS hurdles.